setup_environment.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+###################################################################
+# The purpose of this script is to download, install
+# and configure all files and programs required to
+# take part in developing ActivitySync (Android only).
+###################################################################
+
+CURRENT_USER=$USER
+
+# update package lists from repositories
+sudo apt-get update
+sudo apt-get install git
```

Comment:
git might be already installed. Better to check if present (sudo not needed), only if missing, install it using sudo.